### PR TITLE
Use tag gcr.io/distroless/base-debian11:latest for dnsmasq base image

### PR DESCRIPTION
**Pull request by jingyuanliang**
To alleviate the pain of continuous bumping.

### CODE REVIEW

File: images/dnsmasq/Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ export DOCKER_CLI_EXPERIMENTAL=enabled
 
 # Multiarch image
 # Uploaded: Jun 2, 2023, 12:57:53 PM
```
Comment (DamianSawicki) [inline images/dnsmasq/Makefile:31]: I believe this line should be removed if we are to switch from a specific sha to `latest`.

Comment (jingyuanliang) [reply images/dnsmasq/Makefile]: done

**Comment by bowei** [comment]
Please rebase?

**Pull request closed without merging**